packages/python-packages/api-stub-generator/apistub/_apiview.py

```diff
@@ -25,29 +25,29 @@ class ApiView:
     """Entity class that holds API view for all namespaces within a package
     :param NodeIndex: nodeindex
     :param str: pkg_name
-    :param str: pkg_version
     :param str: ver_string
     """
 
-    def __init__(self, nodeindex, pkg_name="", pkg_version="", namespace = ""):
-        self.Name = pkg_name
-        self.Version = 0
-        self.VersionString = ""
-        self.Language = "Python"
-        self.Tokens = []
-        self.Navigation = []
-        self.Diagnostics = []
+    def __init__(self, nodeindex, pkg_name="", namespace = "", cross_language_map=None):
+        self.name = pkg_name
+        self.version = 0
```

Comment:
Why was version removed from the constructor?

Reply:
Because it wasn't used anywhere.